pgmpy/models/BayesianModel.py

```diff
@@ -315,10 +315,12 @@ def remove_cpds(self, *cpds):
                 cpd = self.get_cpds(cpd)
             self.cpds.remove(cpd)
 
-    def get_cardinality(self, node):
+    def get_cardinality(self, node=None):
```

Comment:
@raghavg7796 Okay. But, I am of the opinion that we should keep the same data type of the return value. Which means it should return a dictionary even if the ```node``` is specified . Similar changes should be made in ```MarkovModel.get_cardinality``` also. What do you think?